FunctionalTableData/CellStyle.swift

```diff
@@ -9,18 +9,41 @@
 import Foundation
 import UIKit
 
+/// The actions property exposed on the CellConfigType represents possible events that will be executed based on the users interaction with that particular cell. Of note are the `selectionAction` and `previewingViewControllerAction`. The `selectionAction` is executed when the user taps on that particular cell. The main use case for this is present a new detail view controller or a modal (but is not constrained to these actions, these are just the common use cases). The `previewingViewControllerAction` is responsible for returning an instance of a UIViewController that will be shown when a user 3D-touches on a cell.
```

Comment:
This comment is a duplicate of the comment for `CellActions`, it does not document `CellStyle`.

Also line breaks are nice 🙈

Also missing apostrophe: `user's interaction`